src/ScottPlot5/ScottPlot5/Primitives/Color.cs

```diff
@@ -116,32 +126,38 @@ public static Color FromARGB(int argb)
 
     public static Color FromARGB(uint argb)
     {
-        byte alpha = (byte)(argb >> 24);
-        byte red = (byte)(argb >> 16);
-        byte green = (byte)(argb >> 8);
-        byte blue = (byte)(argb >> 0);
-        return new Color(red, green, blue, alpha);
+        return new Color(argb);
     }
 
-    public static Color FromHex(string hex)
+    public static Color FromHex(string hex) => new(hex);
+
+    //Returns ARGB value from hex string
+    public static uint Hex2Argb(string hex)
```

Comment:
Maybe it's better to make this method private?

Reply:
oh ya he did mention private in the original post, can make the update.

Reply:
LGTM!